### PR TITLE
added python code

### PR DESCRIPTION
**Pull request by saa938**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [✔] I have tested my code.
- [✔] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [✔] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by ChuangSheep** [comment]
cf. #3196
and just FYI, u don't have to close the PR when u make changes - just push them to the branch

File: solutions/usaco-104.mdx [hunk 1]
```diff
@@ -87,5 +87,26 @@ class haybaleStacking {
 
 </JavaSection>
 	
-</LanguageSection>
+<PythonSection>
+
+## Python Implementation
```
Comment (ryanchou-dev) [inline solutions/usaco-104.mdx:92]: unnecessary

File: solutions/usaco-104.mdx [hunk 2]
```diff
@@ -87,5 +87,26 @@ class haybaleStacking {
...
+	
+```python
+n, k = map(int, input().split())
+differences = [0] * (n+1) # make a differences array
```
Comment (ryanchou-dev) [inline solutions/usaco-104.mdx:96]: space out operators

File: solutions/usaco-104.mdx [hunk 3]
```diff
@@ -87,5 +87,26 @@ class haybaleStacking {
...
+for i in range(k):
+	start, end = map(int, input().split())
+	differences[start - 1] += 1 # add one to the start
+	differences[end] -= 1 # subtract one from the next number outside of start-end
+pref = [0] * n # prefix sums array
```
Comment (ryanchou-dev) [inline solutions/usaco-104.mdx:101]: please split up chunks of code

**Review by ryanchou-dev — CHANGES_REQUESTED** [review]

**Comment by SansPapyrus683** [comment]
fixed everything myself lol

**Comment by saa938** [comment]
thanks man and ill remember to push them to the branch

**Comment by SansPapyrus683** [comment]
can you guys re-review please

**Review by ChuangSheep — APPROVED** [review]
nice

**Pull request merged**